### ISSUE TITLE
isDeveloperMode makes some libraries not loadimg

### ISSUE THREAD

**Issue by eymen-elkum** [espocrm/espocrm#2836]
**Describe the bug**
I enabled Develoer Mode as described [here](https://docs.espocrm.com/development/how-to-start/#configuration-for-development)

**Expected behavior**
Libraries like jquery, bullbone, handlebars, underscore, marked and dompurify to be loaded but they don’t.

**Screenshots**
![Screenshot 2023-09-10 at 5 15 44 PM](https://github.com/espocrm/espocrm/assets/2862528/f5568e44-ff8a-417b-b480-5d58eff28256)
![Screenshot 2023-09-10 at 5 18 13 PM](https://github.com/espocrm/espocrm/assets/2862528/791290b8-5697-46ab-b0b5-193494f02a3a)

**EspoCRM version**
8.0.0 [fix branch]

**Additional context**
I noticed that libs are loading twice, first time loading correctly and second is 404

1- http://localhost/crm-extensions-8.0/client/lib/original/handlebars.js?r=1694354013
2- http://localhost/crm-extensions-8.0/handlebars?_=1694354014404

**Comment by yurikuzn** [comment]
We work in developer mode, no problem. Might need to build.

**Comment by yurikuzn** [comment]
As it's a weekend today, I want to ask everybody, for such issues, first to try to figure out the problem by yourself, then to report it on Monday. I already looked into 4 false bug reports today, it supposed to be my only no-work day.

**Comment by eymen-elkum** [comment]
I reply today to never bother you again on your weekend,

Note I was also working on weekend, I usually dont work on weekends but yesterday I did, that was my decision.

When I post an issue here I (maybe all of us) never expect instant reply.

Why you tell me that you fixed 4 bugs yesterday? when you do that you are growing your business, and your business is really great and success.

We share almost the same conditions and when you tell me such simple information you just like if you tell me that I am very beginner and I have no experience or information about how things goes here.

I am sorry for this answer, I dont want to be annoying here and I do used to fix my issues myself, but all of us can miss something some days.

TLDR;
Just dont reply when you are in weekend, I do this, I believe this is doable and better than telling people that they are annoying.
